plugin-tester-scala/src/main/scala/io/grpc/examples/helloworld/GreeterServer.scala

```diff
@@ -35,7 +35,8 @@ object GreeterServer {
     implicit val mat: Materializer = ActorMaterializer()
     implicit val ec: ExecutionContext = sys.dispatcher
 
-    val service: HttpRequest => Future[HttpResponse] = GreeterServiceHandler(new GreeterServiceImpl(mat))
+    val service = GreeterServiceHandler(new GreeterServiceImpl(mat))
+      .orElse { case _  => Future.successful(HttpResponse(StatusCodes.NotFound)) }
```

Comment:
This previously passed a PartialFunction to akka http meaning that an unmatched route blew up with a MatchError